Documentation: Update README (12.11.2024)

Gero1999 opened the pull request:
## Issue

Closes #110 

## Description

Current README in the main branch is outdated, does not contain all required information and even has some incorrect statements. Should me updated to the current state of the package.

## Definition of Done
Update all relevant content

## How to test
- [x] Installation guidelines work well
- [x] Content information is updated and accurate

`Feel free to modify or add content as well!`

## Contributor checklist
- [ ] Code passes lintr checks
- [ ] Code passes all unit tests
- [ ] New logic covered by unit tests
- [ ] New logic is documented

js3110 reviewed (approved):
LGTM

m-kolomanski commented on README.md:
```diff
@@ -1,91 +1,66 @@
+To ensure a clean codebase and smooth cooperation, please adhere to the [contributing guidelines](CONTRIBUTING.md).
 
+### As user 
+Feel free to open identified issues, to reach out to us for questions or fill our [google sheet]() for feedback.
```
issue: please provide the link to google sheet, if we have any?

m-kolomanski commented on README.md:
```diff
@@ -1,91 +1,66 @@
-
 <!-- 
 
 ## Getting started
```
issue: there is a lot of commented out content, please include whatever is relevant and remove the rest.

m-kolomanski reviewed (changes requested):
Hi, from my side:

- I do not think it is currently useful to install the package directly from github. We cannot run the application like that since the dependencies will not load, we will need to fix that. So after installation, you still have to change the directory to the installation directory, which makes more sense when the repo is cloned directly - I have left only this part in, as I think it makes the most sense
- I have updated the badges, as the links pointed towards `shiny` repository, not `aNCA`
- I have updated links to contributing guidelines
- I have added a **Quick start* - so the basics of app usage and what the user can do to test it out upon first startup. Please read and verify that it is correct, maybe some other workflow makes more sense - then please change.

In addition, I have some comments and requested changes. The contents seem to be copied over from a template, please make sure only applicable parts are included and that the links point in the right directions.

Gero1999 commented:
> Hi, from my side:
> 
> * I do not think it is currently useful to install the package directly from github. We cannot run the application like that since the dependencies will not load, we will need to fix that. So after installation, you still have to change the directory to the installation directory, which makes more sense when the repo is cloned directly - I have left only this part in, as I think it makes the most sense
> * I have updated the badges, as the links pointed towards `shiny` repository, not `aNCA`
> * I have updated links to contributing guidelines
> * I have added a *_Quick start_ - so the basics of app usage and what the user can do to test it out upon first startup. Please read and verify that it is correct, maybe some other workflow makes more sense - then please change.
> 
> In addition, I have some comments and requested changes. The contents seem to be copied over from a template, please make sure only applicable parts are included and that the links point in the right directions.

Hello Mateusz! You are right, our main branch has commented code that I was not sure if to keep it perhaps for the future, but I just deleted it and solve the issues you mentioned. I will test with some  others non-programming based people to check if they manage to understand completely the instructions on their own, as I know that sometimes this can be a game changer for our users. 

And many thanks for making the changes, I think now it looks much better! 😎

m-kolomanski reviewed (approved):
I agree with @Gotfrid , changing how the package loads dependencies is out of scope of this issue and that is why I have suggested removing the information altogether. Lets create a separate issue as @Gotfrid suggests, and then update README alongside when that option is available.
